src/sage/rings/real_double.pyx

```diff
@@ -664,18 +666,19 @@ cdef class RealDoubleField_class(sage.rings.abc.RealDoubleField):
         The implementation relies on the ``roots()`` method which often reports
         roots not to be real even though they are::
 
-            sage: f = (x-1)^3
-            sage: f.roots(ring=CDF)  # abs tol 2e-5
+            sage: f = (x-1)^3                                                           # needs numpy
+            sage: f.roots(ring=CDF)  # abs tol 2e-5                                     # needs numpy
             [(1.0000065719436413, 1),
```

Comment:
Why numpy here and below?

Reply:
This goes through `Polynomial.root`, which tries to invoke numpy for low-precision float fields. (It could fall back to using PARI, but this would require a bit of refactoring)

Reply:
OK.